tests/docker_mcp/test_mcp.py

```
@@ -1,1304 +1,598 @@
+        )
+        payload = parse_tool_result(response)
+        assert_condition(payload["state"] == "exited", "quick task should exit", payload)
+        assert_condition(payload.get("pid") is None, "quick exit should not report pid", payload)
+        assert_condition(payload.get("exit_code") == 0, "quick exit should report exit_code 0", payload)
+        assert_condition(
+            "Test task executed successfully" in payload["initial_output"],
+            "quick exit missing task output",
+            payload,
+        )
+        assert_condition(
+            all("ok" not in item for item in payload.keys()),
```

Ruff (0.15.11)

[warning] Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)